### PR TITLE
Use a pinned pipenv to run flake8 in Travis

### PR DESCRIPTION
**Pull request by stephendonner**
Passing ad-hoc here: https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/pollbot.adhoc/6/console

### CODE REVIEW

**Comment by stephendonner** [comment]
I ran out of time, and I've got Python-version (2 vs 3) issues to sort out.  Added flake8 to the Pipfile and tried to generate the lock file, but it bombed out due to my local setup 👎

**Comment by stephendonner** [comment]
If I'm reading and understanding it correctly, it looks like the SHA hashes for dependencies was bumped due to the index changing, which makes sense (cool)!

Passing ad-hoc here: https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/pollbot.adhoc/7/console

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,10 @@ language: python
 python:
   - 3.6
 install:
-  - pip install flake8
+  - pip install -r pipenv.txt
 script:
-  - flake8 .
+  - pipenv check
+  - pipenv sync
```
Comment (davehunt) [inline .travis.yml]: I think this should be `pipenv install` rather than `pipenv sync`

**Review by davehunt — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (stephendonner) [reply .travis.yml]: Thanks; fixed!

**Pull request merged**